inbm-lib/inbm_lib/trtl.py

```diff
@@ -538,6 +538,19 @@ def delete_snapshot(self, snapshot: str) -> Tuple[int, Optional[str]]:
                 out, err, code))
         return code, err
 
+    def delete_config(self, config_filename: str) -> Tuple[int, Optional[str]]:
+        """Trtl wrapper to delete the configuration for a filesystem .
```

Comment:
I'm finding the docstring for this function to be confusing. It should probably mention that it's deleting a snapper config. But this doesn't quite fit either because the parameter is called `config_filename` when snapper refers to configs by name, not filename.